test/utils/link_creator.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+rm -f uri_dump;
+
+# Enable logging in config
+sed -i.bak 's/dump_test_uris:\ false/dump_test_uris:\ true/' config.test.yaml
```

Comment:
I don't like this. If something goes wrong during the script's execution, we end up with a modified `config.test.yaml` and will need manual intervention.